cmd/tailscale/cli/serve.go

```diff
@@ -67,7 +67,7 @@ EXAMPLES
     $ tailscale serve https:8080 / text:"Hello, world!"
 
   - To forward raw TCP packets to a local TCP server on port 5432:
-    $ tailscale serve tcp:2222 tcp://localhost:22
+    $ tailscale serve tcp:2222 tcp://localhost:5432
```

Comment:
Not sure if the 2222 should also be called out in the text as the incoming port.

Reply:
The line before is also slightly misleadingly wrong. It's not forwarding "raw TCP packets"; it's reconstruction a new TCP stream and proxying bytes between them.

"To forward incoming TCP connections to a local TCP server on ..."

Reply:
Should the `tls-terminated-tcp` option below get a similar change?

Reply:
yup

Reply:
Done.